src/components/ebay-pagination/index.js

```diff
@@ -91,46 +92,42 @@ function refresh() {
     for (let i = 0; i < this.state.items.length; i++) {
         if (this.state.items[i].current) {
             current = i;
-        } else {
-            // remove all hidden attribues to get accurate widths
-            this.pageEls[i].removeAttribute('hidden');
         }
+        this.pageEls[i].removeAttribute('hidden');
```

Comment:
Did you feel that the comment wasn't needed for this?

Reply:
I can add one back in if needed.